### PR TITLE
BP-2822: QES changes and general improvements

### PR DESCRIPTION
**Pull request by bauer00**

### CODE REVIEW

**Review by alnicoli — APPROVED** [review]

File: packages/verification-vue-component/src/locales/fr.json [hunk 1]
```diff
@@ -61,26 +61,42 @@
...
                 "smartContractAddress": "Adresse du Smart Contract",
                 "signatureTransactions": "Transactions de signature",
                 "details": "Détails",
-                "initiator": "Initiateur"
+                "initiator": "Initiateur",
+                "name": "Name",
+                "email": "E-Mail",
+                "phone": "Phone",
+                "company": "Company"
+            },
```
Comment (bauer00) [inline packages/verification-vue-component/src/locales/fr.json:81]: @emeka could you please translate the texts which are in english?

File: packages/verification-vue-component/src/locales/it.json [hunk 2]
```diff
@@ -61,26 +61,42 @@
...
                 "revocationDate": "Data e ora di annullamento",
                 "smartContractAddress": "Indirizzo del Smart Contract",
                 "signatureTransactions": "Transazioni di firma",
                 "details": "Dettagli",
-                "initiator": "Iniziatore"
+                "initiator": "Iniziatore",
+                "name": "Name",
+                "email": "E-Mail",
+                "phone": "Phone",
+                "company": "Company"
```
Comment (bauer00) [inline packages/verification-vue-component/src/locales/it.json:80]: @ianaz could you please translate the texts which are still in english?

File: packages/verification-vue-component/src/locales/it.json [hunk 3]
```diff
@@ -61,26 +61,42 @@
                 "contract": "Contratto",
                 "issuer": "Emittente",
                 "signature": "Firma | Firme",
-                "issuerVerifiedBy": "Emittente verificato da",
+                "verifiedBy": "Verificato da",
+                "signedAt": "Signed at",
```
Comment (ianaz) [inline packages/verification-vue-component/src/locales/it.json:65]:
```suggestion
                "signedAt": "Firmato il",
```

File: packages/verification-vue-component/src/locales/it.json (hunk 2 above)
Comment (ianaz) [inline packages/verification-vue-component/src/locales/it.json:80]:
```suggestion
                "name": "Name",
                "email": "E-Mail",
                "phone": "Telefono",
                "company": "Azienda"
```

File: packages/verification-vue-component/src/locales/it.json [hunk 4]
```diff
@@ -61,26 +61,42 @@
...
+                "company": "Company"
+            },
+            "signature": {
+                "level": {
+                    "AES": {
+                        "title": "Blockchain eSignature",
+                        "description": "Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua."
+                    },
+                    "QES": {
+                        "title": "Blockchain eSignature + Qualified Electronic Signature (CH)",
```
Comment (ianaz) [inline packages/verification-vue-component/src/locales/it.json:89]:
```suggestion
                        "title": "Blockchain eSignature + Firma Elettronica Qualificata (CH)",
```

File: packages/verification-vue-component/src/locales/fr.json (hunk 1 above)
Comment (bauer00) [reply packages/verification-vue-component/src/locales/fr.json:80]: @emeka reminder, also added "signatureType" which I missed before.

File: packages/verification-vue-component/src/locales/it.json [hunk 5]
```diff
@@ -56,31 +56,46 @@
                 "hide": "Nascondere info avanzate ",
                 "show": "Info avanzate"
             },
-            "issuer": "Emittente",
             "meta": {
-                "contract": "Contratto",
                 "issuer": "Emittente",
                 "signature": "Firma | Firme",
-                "issuerVerifiedBy": "Emittente verificato da",
+                "signatureType": "Signature type",
```
Comment (bauer00) [inline packages/verification-vue-component/src/locales/it.json:62]: @ianaz missed this one 😊

Comment (ianaz) [inline packages/verification-vue-component/src/locales/it.json:62]:
```suggestion
                "signatureType": "Tipo di firma",
```
:)

Comment (bauer00) [reply packages/verification-vue-component/src/locales/it.json:62]: Thanks

**Comment by alnicoli** [comment]
lgtm

**Pull request merged**